src/constants/CardPropTypes.js

```diff
@@ -295,6 +296,47 @@ export const BarChartCardPropTypes = {
   tooltipDateFormatPattern: PropTypes.string,
 };
 
+export const PieCardPropTypes = {
+  /** Configuration content for the PieChart */
+  content: PropTypes.shape({
+    /** Object that maps the groupDataSourceId with a specific color, e.g. {'Group A': 'red', 'Group B': 'blue', ... } */
+    colors: PropTypes.object,
+    /** Function to output custom HTML for the tooltip. Passed an array or object with the data,
+     * and the default tooltip markup */
+    customTooltip: PropTypes.func,
+    /** Defaults to 'group' but can be overriden to be any string property of the objects in the values array */
+    groupDataSourceId: PropTypes.string,
+    /**
+     * Function to format the labels. Input to the function is a wrapped data object containing additional
+     * chart label info such as x & y positions etc */
+    labelsFormatter: PropTypes.func,
+    /** The position of the legend in relation to the chart, can be 'bottom' or 'top'. */
+    legendPosition: PropTypes.string,
+  }),
+  /** Used to overide the internal components and props for advanced customisation */
+  overrides: PropTypes.shape({
```

Comment:
Is this the new standard for our components? Seems like it opens up the gates to too much freedom

Reply:
Probably not a new standard for every component, but perhaps an escape hatch to be used/provided if needed. For the PieChartCard we did not have much of a spec so I thought it would make sense to use this pattern here. We have already used it for a couple of components.